docs/Permissions/Onchain-Permissioning/Onchain-Permissioning.md

```diff
@@ -0,0 +1,51 @@
+description: Onchain Permissioning
+<!--- END of page meta data -->
+
+# Onchain Permissioning 
+
+Onchain permissioning uses smart contracts to store and maintain the node, account, and admin account whitelists. 
+Using onchain permissioning enables all nodes to read the whitelists from one source.                                        
+
+## Permissioning Contracts
+
+The permissioning smart contracts are provided in the [PegaSysEng/permissioning-smart-contracts](https://github.com/PegaSysEng/permissioning-smart-contracts) repository: 
+
+* Ingress contracts for nodes and accounts - proxy contracts that defer the permissioning logic to the 
+Node Rules and Account Rules contracts. The Ingress contracts are deployed to static addresses. 
+
+* Node Rules - stores the node whitelist and node whitelist operations (for example, add and remove). 
+
+* Account Rules - stores the accounts whitelist and account whitelist operations (for example, add and remove). 
+
+* Admin - stores the list of admin accounts and admin list operations (for example, add and remove). There is 
+one list of admin accounts for node and accounts.
+
+## Permissioning Management Dapp
+
+The [Permissioning Management Dapp](Getting-Started-Onchain-Permissioning.md) is provided to view 
+and maintain the whitelists. 
+
+!!! tip 
+    Before v1.2, we provided a [management interface using Truffle](https://docs.pantheon.pegasys.tech/en/1.1.4/Permissions/Onchain-Permissioning/).
+    The management interface using Truffle is deprecated and we recommend using the Dapp for an improved user experience. 
```

Comment:
Can we add emphasis to *deprecated*?

Reply:
We don't typically add emphasis unless it's too a button, tab, etc.   Too much emphasis makes the text harder to scan/read.